pdf/core/encoding.go

```diff
@@ -1897,8 +1898,8 @@ func (enc *CCITTFaxEncoder) DecodeBytes(encoded []byte) ([]byte, error) {
 		EndOfBlock:             enc.EndOfBlock,
 		BlackIs1:               enc.BlackIs1,
 		DamagedRowsBeforeError: enc.DamagedRowsBeforeError,
-		Rows:                   enc.Rows,
-		EncodedByteAlign:       enc.EncodedByteAlign,
+		Rows:             enc.Rows,
```

Comment:
goimports on Windows seems to work differently than on Linux with regards to this indentation